src/workspace.coffee

```diff
@@ -612,16 +612,19 @@ class Workspace extends Model
 
   # Increase the editor font size by 1px.
   increaseFontSize: ->
+    @originalFontSize ?= atom.config.get("editor.fontSize")
```

Comment:
Thinking about this more, if I change the value from the settings view and then run the reset command, it won't reset it right?